project-management/src/main/java/life/qbic/projectmanagement/application/measurement/validation/MeasurementProteomicsValidator.java

```diff
@@ -90,81 +89,51 @@ public static Collection<String> properties() {
     return Arrays.stream(PROTEOMICS_PROPERTY.values()).map(PROTEOMICS_PROPERTY::label).toList();
   }
 
-  @Override
   @PreAuthorize("hasPermission(#projectId, 'life.qbic.projectmanagement.domain.model.project.Project', 'WRITE')")
-  public ValidationResult validate(ProteomicsMeasurementMetadata measurementMetadata,
+  public ValidationResult validateRegistration(MeasurementRegistrationInformationPxP metadata,
+      String experimentId,
       ProjectId projectId) {
     var validationPolicy = new ValidationPolicy();
-    //We want to fail early so we check first if all the mandatory fields were filled
-    ValidationResult mandatoryValidationResult = validationPolicy.validateMandatoryDataProvided(
-        measurementMetadata);
-    if (mandatoryValidationResult.containsFailures()) {
-      return mandatoryValidationResult;
+    var result = ValidationResult.successful();
+
+    for (String sampleId : metadata.measuredSamples()) {
+      result = result.combine(validationPolicy.validationProjectRelation(sampleId, projectId))
+          .combine(
+              validationPolicy.validationExperimentRelation(sampleId, experimentId, projectId));
     }
-    //If all fields were filled then we can validate the entries individually
-    return validationPolicy.validateSampleId(measurementMetadata.sampleCode())
-        .combine(validationPolicy.validateMandatoryDataProvided(measurementMetadata))
-        .combine(validationPolicy.validateOrganisation(measurementMetadata.organisationId())
-            .combine(validationPolicy.validateMsDevice(measurementMetadata.msDeviceCURIE())));
-  }
 
-  @PreAuthorize("hasPermission(#projectId, 'life.qbic.projectmanagement.domain.model.project.Project', 'WRITE')")
-  public ValidationResult validateRegistration(MeasurementRegistrationInformationPxP metadata,
-      ProjectId projectId) {
-    var validationPolicy = new ValidationPolicy();
     ValidationResult mandatoryValidationResult = validationPolicy.validateMandatoryDataProvided(
         metadata);
     if (mandatoryValidationResult.containsFailures()) {
       return mandatoryValidationResult;
     }
 
-    var validation = validationPolicy.validateSampleIdsAsString(metadata.measuredSamples(),
-            projectId)
-        .combine(validationPolicy.validateMandatoryDataProvided(metadata))
-        .combine(validationPolicy.validateOrganisation(metadata.organisationId())
-            .combine(validationPolicy.validateMsDevice(metadata.msDeviceCURIE())));
-
     // For the sample-specific metadata
     var missingLabelsValidation = new MissingLabel(() -> metadata).execute();
     var distinctLabelsValidation = new HasDistinctLabels(() -> metadata).execute();
 
-    return validation.combine(missingLabelsValidation)
+    return result.combine(
+            validationPolicy.validateSampleIdsAsString(metadata.measuredSamples(), projectId))
+        .combine(validationPolicy.validateMandatoryDataProvided(metadata))
```

Comment:
again duplicate mandatory data provided validation. One in the last chain is enough